frontend/src/features/projects/list/ProjectListView.tsx

```diff
@@ -43,73 +43,42 @@ interface IProjectFilterState {
   name?: string;
   statusId?: string;
   agencyId?: string;
-  assessWorkflow?: boolean;
-  agencies?: number;
-  fiscalYaer?: number;
+  agencies?: number | null;
+  fiscalYear?: number | null;
 }
 
-const initialValues = {
+interface IProps {
+  filterable?: boolean;
+  title: string;
+  defaultFilter?: IProjectFilterState;
```

Comment:
A new prop `defaultFilter` allows initializing the project list view appropriately (specifically for approval requests page).